src/generic/data/thunks.js

```diff
@@ -49,3 +53,26 @@ export function updateCreateOrRerunCourseQuery(courseData) {
     }
   };
 }
+
+export function fetchUserPermissionsQuery(courseId) {
+  return async (dispatch) => {
+    try {
+      const userPermissions = await getUserPermissions(courseId);
+      dispatch(updateUserPermissions(userPermissions));
+    } catch (error) {
+      console.trace({ error }); // eslint-disable-line no-console
```

Comment:
Does it make sense for the user to see the permission fetch error? If yes, replace the `console.trace` with a dispatch that updates the errors for the page.

Reply:
We don't need to show any error for now, I changed this to same pattern as other methods in `thunk` (updateLoadingStatus).